wildfly-mixin/dashboards/wildfly-overview.libsonnet

```
@@ -0,0 +1,711 @@
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local serverLogsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{job="integration/wildfly"} |= ``',
+      queryType: 'range',
+      refId: 'A',
+    },
+  ],
+  type: 'logs',
+  title: 'Server logs',
+  description: 'Recent logs from server log file',
+  options: {
+    dedupStrategy: 'none',
+    enableLogDetails: true,
+    prettifyLogMessage: false,
+    showCommonLabels: false,
+    showLabels: false,
+    showTime: false,
+    sortOrder: 'Descending',
+    wrapLogMessage: false,
+  },
+  pluginVersion: '9.1.7',
+};
+
+local sessionsRow = {
+  datasource: promDatasource,
+  targets: [],
+  type: 'row',
+  title: 'Sessions',
+  collapsed: false,
+};
+
+local activeSessionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'wildfly_undertow_active_sessions{deployment="$deployment", job="$job", instance="$instance"}',
+      datasource=promDatasource,
+      legendFormat='{{deployment}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Active Sessions',
+  description: 'Number of active sessions to deployment over time',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local expiredSessionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(wildfly_undertow_expired_sessions_total{deployment="$deployment", job="$job", instance="$instance"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{deployment}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Expired Sessions',
```

```suggestion
  title: 'Expired sessions',
```

All panels are sentence case.